kasa/tests/test_aestransport.py

```diff
@@ -57,7 +58,7 @@ async def test_handshake(
 ):
     host = "127.0.0.1"
     mock_aes_device = MockAesDevice(host, status_code, error_code, inner_error_code)
-    mocker.patch.object(httpx.AsyncClient, "post", side_effect=mock_aes_device.post)
+    mocker.patch.object(aiohttp.ClientSession, "post", side_effect=mock_aes_device.post)
```

Comment:
As we now have an abstraction layer for HTTP requests, I'm wondering if we should rather patch its internals?